app/app_module_config.go

```diff
@@ -84,20 +114,18 @@ var (
 		// We allow the following module accounts to receive funds:
 		// govtypes.ModuleName
 	}
+)
 
-	// application configuration (used by depinject)
-	AppConfig = appconfig.Compose(&appv1alpha1.Config{
+// GetAppConfig returns the depinject config for building modules
+func GetAppConfig() depinject.Config {
```

Comment:
Turn it to function so that `sdk.Config` can be set before initializing app config, or `desmos` bench32 prefix can not be set properly in `x/auth` since var variable will initialize first.

References: `https://medium.com/@denniswon/the-go-init-function-14358757f2e1`